pkg/crc/preflight/preflight_darwin.go

```diff
@@ -33,19 +48,36 @@ func StartPreflightChecks() {
 }
 
 // SetupHost performs the prerequisite checks and setups the host to run the cluster
-func SetupHost() {
+func SetupHost(vmDriver string) {
 	preflightCheckAndFix(false,
 		checkOcBinaryCached,
 		fixOcBinaryCached,
 		"Caching oc binary",
 		false,
 	)
-	preflightCheckAndFix(config.GetBool(cmdConfig.SkipCheckVirtualBoxInstalled.Name),
-		checkVirtualBoxInstalled,
-		fixVirtualBoxInstallation,
-		"Setting up virtualization",
-		config.GetBool(cmdConfig.WarnCheckVirtualBoxInstalled.Name),
-	)
+
+	switch vmDriver {
+	case "virtualbox":
+		preflightCheckAndFix(config.GetBool(cmdConfig.SkipCheckVirtualBoxInstalled.Name),
+			checkVirtualBoxInstalled,
+			fixVirtualBoxInstallation,
+			"Setting up virtualization",
```

Comment:
I suggest to use the hypervisor name here too, to avoid ambiguity

Reply:
Fixed locally, this will be in the next update to this PR